apps/src/studio/studio.js

```diff
@@ -2478,13 +2523,60 @@ Studio.createLevelItems = function (svg) {
   }
 };
 
-Studio.drawMapTiles = function () {
+Studio.isWallTile = function(row, col) {
```

Comment:
This function (and getWallTile below) behave differently from the new `getWallValue` function. We should modify them to be consistent.